flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/CastFunctionITCase.java

```diff
@@ -945,12 +945,11 @@ protected Configuration configuration() {
                                 "2021-09-27 12:34:56.123",
                                 fromLocalToUTC(
                                         LocalDateTime.of(2021, 9, 27, 12, 34, 56, 123000000)))
-                        // https://issues.apache.org/jira/browse/FLINK-24446 Fractional seconds are
-                        // lost
                         .fromCase(
                                 STRING(),
                                 "2021-09-27 12:34:56.123456789",
-                                fromLocalToUTC(LocalDateTime.of(2021, 9, 27, 12, 34, 56, 0)))
+                                fromLocalToUTC(
+                                        LocalDateTime.of(2021, 9, 28, 22, 52, 32, 789000000)))
```

Comment:
May be a stupid question however could you please clarify why in String value there is `123456789` as fractions of seconds and in target `789000000` ?